lib/osf-components/addon/components/maintenance-banner/component.ts

```diff
@@ -54,6 +67,18 @@ export default class MaintenanceBanner extends Component.extend({
     }
 
     didReceiveAttrs(this: MaintenanceBanner): void {
-        this.get('getMaintenanceStatus').perform();
+        if (!this.cookies.exists(maintenanceCookie)) {
+            this.get('getMaintenanceStatus').perform();
+        }
+    }
+
+    @action
+    dismiss(this: MaintenanceBanner) {
+        this.cookies.write(maintenanceCookie, 0, {
+            expires: moment().add(24, 'hours').toDate(),
+            path: '/',
+            domain: cookieDomain,
```

Comment:
Oh whoops, the last two times I've added a cookie write I haven't set the domain. Should I go back and fix those?

Reply:
I'm not positive. Check the cookies that the legacy app makes and see if it has a domain, but I'm reasonably sure we want one for the cookies.